pynicotine/gtkgui/frame.py

```diff
@@ -2212,7 +2211,7 @@ def update_log(self, msg, level):
             message_dialog(parent=self.application.get_active_window(), title=title, message=msg)
             return
 
-        if config.sections["logging"]["logcollapsed"]:
+        if level not in ("connection", "message", "miscellaneous"):
```

Comment:
A comment about why specific levels are disabled would be nice to have.

Reply:
Sorry, should've been more specific. A short code comment above this line telling someone who browses the code why this edge case is here.

Reply:
Omg you want me to put comments in the code?! I hope users don't mind the installation being > 1GB :-D